### PR TITLE
[realsense2] Add tools compilation

### PR DESCRIPTION
**Pull request by claudiofantacci**
With this PR the feature `[tools]` is added to `realsense2` port.
Realsense tools are listed [here](https://github.com/IntelRealSense/librealsense/tree/master/tools) and they include viewer, configuration and debugging applications.

Tools are built and moved to `tools` directory in `vcpkg` dir tree.

I also reordered some CMake commands in the `portfile.cmake` trying to make a bit of order in the function calls.

### CODE REVIEW

File: ports/realsense2/CONTROL [hunk 1]
```diff
@@ -3,4 +3,8 @@ Version: 2.10.1
 Description: Intel® RealSense™ SDK 2.0 is a cross-platform library for Intel® RealSense™ depth cameras (D400 series and the SR300).
```
Comment (UnaNancyOwen) [inline ports/realsense2/CONTROL]: Please increment port version.
```patch
- Version: 2.10.1
+ Version: 2.10.1-1
```

Comment (claudiofantacci) [reply ports/realsense2/CONTROL]: Thanks for spotting this!
I've incrmented the version number following your suggestion.

**Comment by UnaNancyOwen** [comment]
@ras0219-msft @alexkaratarakis Could you merge this pull request? Thnaks,

**Comment by ras0219-msft** [comment]
Thanks for the PR and sorry for the delay in merging!

**Pull request merged**